vernac/g_vernac.mlg

```diff
@@ -1170,8 +1170,10 @@ GRAMMAR EXTEND Gram
       | IDENT "Visibility"; s = OPT IDENT -> { PrintVisibility s }
       | IDENT "Implicit"; qid = smart_global -> { PrintImplicit qid }
       | b = [ IDENT "Sorted" -> { true } | -> { false } ]; IDENT "Universes";
-        g = OPT printunivs_subgraph; fopt = OPT ne_string ->
-        { PrintUniverses (b, g, fopt) }
+        g = OPT printunivs_subgraph;
+        with_sources = OPT [ b = [ "With" -> { true } | "Without" -> { false } ]; "Constraint"; "Sources" -> { b } ];
```

Comment:
Note to self: most of the CI errors are because these idents are unecessarily made keywords, will fix

coq-lsp needs an overlay 

The refman doesn't like the `coq:: reset` I added

Reply:
> The refman doesn't like the `coq:: reset` I added

The definition of pidentity is before the reset (line 498), so it's gone when referenced (line 687).